wiki/Ranking_criteria/Simplified_ranking_criteria/en.md

```diff
@@ -0,0 +1,181 @@
+# Simplified ranking criteria
+
+*For full ranking criteria, see: [ranking criteria](/wiki/Ranking_criteria)*
+
+The [ranking criteria](/wiki/Ranking_criteria) is complicated. It explains every rule and guideline required to create a [beatmap](/wiki/Beatmap) worthy of being [Ranked](/wiki/Beatmap_ranking_procedure#ranked), including *a lot* of niche situations that the majority of mappers never encounter.
+
+The **simplified ranking criteria** aims to give mappers a clearer perspective on what's required to create a rankable map through the following:
+
+- Simplified rules and guidelines that impact most beatmaps
+- Subjective criteria that mappers encounter when ranking their maps
+
+## Beatmap
+
+::: Infobox
+![](./img/percent.png "Song progress percentage")
+:::
+
+- **Maps must be 30 seconds long.**
+- **Maps must end after the [80% mark](./img/percent.png).** If you want to end the map earlier, cut the song.
```

Comment:
this is a guideline that still gets justifiably broken nowadays, so i'm not sure if it should be worded as strongly as this